rqd/rqd/rqcore.py

```diff
@@ -98,10 +98,7 @@ def __createEnvVariables(self):
         self.frameEnv["CUE_GPU_MEMORY"] = str(self.rqCore.machine.getGpuMemoryFree())
         self.frameEnv["SP_NOMYCSHRC"] = "1"
 
-        if platform.system() in ("Linux", "Darwin"):
-            self.frameEnv["MAIL"] = "/usr/mail/%s" % self.runFrame.user_name
-            self.frameEnv["HOME"] = "/net/homedirs/%s" % self.runFrame.user_name
-        elif platform.system() == "Windows":
```

Comment:
These variables are spi specific and safe to be removed. As these env vars are required at SPI, I think this PR should provide a alternative to how to set these variables from now on. I guess the following logic can be used, and it should be documented on this PR.

```python
 for key, value in self.runFrame.environment.items():
            if key == 'PATH':
                self.frameEnv[key] += os.pathsep + value
            else:
                self.frameEnv[key] = value
```

Reply:
rqd.conf already have a way of passing host enviroment variables onto the job (under [UseHostEnvVar])

Reply:
Have added how to re-enable them again in the PR description